runner/src/main.rs

```diff
@@ -22,6 +22,10 @@ pub struct Cli {
     /// Whether to output verbose debugging information to the console.
     #[clap(long, short)]
     verbose: bool,
+
+    /// The architecture of the elf file
+    #[clap(long, short)]
+    architecture: Option<String>,
```

Comment:
Why does this need to be passed on the command line? Reading `LIBTOCK_PLATFORM` should be sufficient.

Reply:
I think you have a point, there is no reason why users would specify another architecture.